### DISCUSSION TITLE
Does Hyperswitch support Apple Store subscription for ios client

### DISCUSSION THREAD

**Discussion by Stellachen11** [juspay/hyperswitch#4638] [Q&A]
Does Hyperswitch support Apple Store subscription for ios client?

We are now directly connected to Apple's subscription capabilities in Apple's technical documentation, including various discount capabilities, which feels very troublesome. Do you now support Apple subscriptions for clients? Users only trigger active payment for the first subscription, and Apple will automatically deduct the fee for each subsequent period.

**Comment by tinu-hareesswar** [comment]
Hi @Stellachen11, we support recurring payments on Apple Pay by setting up an MIT which the user has to authorize during the first payment. Post which you can you the hyperswitch payment_method_id to deduct the fee for each period. You can refer top the docs here to learn how to set it up -  https://docs.hyperswitch.io/features/payment-flows-and-management/mandates-and-recurring-payments

If this is not solving for your use case, could you please elaborate further on it and share some Apple documentation or UI screens for us to understand it better.
